lib/drawSquareDoubles.c

```diff
@@ -14,6 +13,95 @@
 #define DEBUG2 '2'
 #define DEBUG3 '3'
 
+triangle* meshInit (int numberOfTris){
+    return (triangle*) malloc(numberOfTris * sizeof(triangle));
+}
+
+void plotTrianglePoints(double arr[][3], triangle* triangles, int numberOfTris){
+
+    int totalNumberOfPoints = numberOfTris*3;
+    int count = 0;
+
+    for (int i = 0; i < numberOfTris; i++){
+        for(int j = 0; j < 3; j++){
+                
+            arr[count][0] = triangles[i].p[j].x;
+            arr[count][1] = triangles[i].p[j].y;
+            arr[count][2] = triangles[i].p[j].z;
+            count++;    
+        }
+    }
+}
+
+void projectTrianglePoints2d(double arr[][3], double p_points[][2], const double DISTANCE, int iter){
```

Comment:
func name is plot tirangle points, but this function doesnt know anything about triangles, it is given an array of points.

Reply:
Was kind of copying functions from previous method, thought I would have equivalent functions, but they ended up being quite different in nature. Function names can definitely be improved.